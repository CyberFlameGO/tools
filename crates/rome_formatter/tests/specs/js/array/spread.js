let a = [...a, ...b,];
let b = [...a, ...b];
[...d]
let x = [...baaaaaaaaaaaaaaaaaaaaaaaaaaaaa,...bbbbbbbbbbbbbbbbbbbbbbbbbbbbbb,...cccccccccccccccccccccccccccccc]

