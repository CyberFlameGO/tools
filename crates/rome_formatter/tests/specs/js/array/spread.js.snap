---
source: crates/rome_formatter/tests/spec_test.rs
assertion_line: 43
expression: spread.js

---
# Input
let a = [...a, ...b,];
let b = [...a, ...b];
[...d]
let x = [...baaaaaaaaaaaaaaaaaaaaaaaaaaaaa,...bbbbbbbbbbbbbbbbbbbbbbbbbbbbbb,...cccccccccccccccccccccccccccccc]


---
# Output
let a = [...a, ...b];
let b = [...a, ...b];
[...d];
let x = [
	...baaaaaaaaaaaaaaaaaaaaaaaaaaaaa,
	...bbbbbbbbbbbbbbbbbbbbbbbbbbbbbb,
	...cccccccccccccccccccccccccccccc,
];

